### PR TITLE
Install pip from cwl-ica conda env by exporting path first

### PR DESCRIPTION
**Pull request by alexiswl**
Added to debug investigation of pip setuptools error

### CODE REVIEW

**Comment by alexiswl** [comment]
Will revert back and use single quotes

File: .github/scripts/run_sync-tools_and_sync-workflows.sh [hunk 1]
```diff
@@ -65,8 +65,9 @@ conda run \
 # Install setuptools from pip
 echo "## DEBUG Forcing installation of setuptools to less than version 58 which seems to be causing some errors" 1>&2
 conda run \
- --name "${CONDA_ENV_NAME}" \
- bash -c "pip install setuptools<58"
+  --name "${CONDA_ENV_NAME}" \
+  bash -c 'pip install setuptools<58'
```
Comment (scwatts) [inline .github/scripts/run_sync-tools_and_sync-workflows.sh:69]: The command responsible for triggering is:
```
conda run ['bash', '-c', 'pip install setuptools<58']
```

But needs to read:
```
conda run ['bash', '-c', 'pip install "setuptools<58"']
```

So the quoting needed here is:
```
bash -c 'pip install "setuptools<58"'
```

**Review by scwatts — CHANGES_REQUESTED** [review]

**Review by scwatts — APPROVED** [review]

**Comment by alexiswl** [comment]
I think I'll just build a container for cwl-ica instead. Looks like it takes six minutes to even just install the basics. Could be useful from user endpoint too.

**Review by scwatts — APPROVED** [review]

**Comment by alexiswl** [comment]
Thanks for your help debugging this Stephen, setuptools in conda env has worked and have created a docker image for amd and arm users too (which strips down a lot of the GH actions workflow too). Tested on separate ec2 instance and sync runs fine! Documentation on using container locally still to come. Will also deploy new release after this merge.

**Pull request merged**